README.md

```diff
@@ -614,8 +616,196 @@ let sameFont = UIFont(font: FontFamily.Helvetica.Bold, size: 16.0) // iOS
 let sameFont = NSFont(font: FontFamily.Helvetica.Bold, size: 16.0) // OS X
 ```
 
----
+## Models
+
```

Comment:
See comment above. I'd rename this to `JSON`, and a general description about how a user could parse whatever they want using custom templates. Explain that there's a `spec` variable.

Then add a subsection about model-parsing.